### ISSUE TITLE
🐛 Bug Report: Recreate Same Name Relationship Attribute Only Self-Hosted

### ISSUE THREAD

**Issue by SafakB** [appwrite/appwrite#8116]
### 👟 Reproduction steps

The error occurs after deleting the existing relationship attribute and adding it again with the same name.

**Relationship Settings**
- One-way Relationshop
- Many to one
- Set NULL

Steps
```
1 - Create Relationship
2 - Data entry with relationship
3 - Delete Relationship attribute
4 - Re-add Relationship
5 - Added to the list but “Failed”
6 - When I check the data I see that the old attributes are still selected :)
```

I found this
```After step 3, in the background the attribute is deleted from the list, but the attribute values remain in the rows```

### 👍 Expected behavior

Deleted relationship should be able to be added again with the same name

### 👎 Actual Behavior

After adding a relationship, it gives a “Failed” warning. The process is not completed.
Error only occurs in Self-Hosted

Our Live Server 1.5.5    [ ❌ Not working]
Our Test Server 1.5.5    [ ❌ Not working]
Zero installation 1.5.5  [ ❌  Not working]
Cloud [ ✅  Working]

![image](https://github.com/appwrite/appwrite/assets/23029364/503939a5-4f99-4c70-9af0-be6608c62aaf)

### 🎲 Appwrite version

Version 1.5.x

### 💻 Operating system

Linux
Windows 10 (Edited)

### 🧱 Your Environment

Used Appwite 1.5.5
The bug is not dependent on any SDK

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by SafakB** [comment]
Also tested on Windows 10. Bug causes problems here too

**Comment by Tanu1201** [comment]
Hey, I am Tanu Sharma having 2 number of exp in backend development, can I take this issue up.

**Comment by stnguyen90** [comment]
@safwanyp, thanks for creating this issue! 🙏

Let's continue troubleshooting in https://discord.com/channels/564160730845151244/1239545933839073301

**Comment by SafakB** [comment]
I'll be sharing a status update

When I check via MariaDB, I see that the column has not really been deleted

![image](https://github.com/appwrite/appwrite/assets/23029364/6424537a-3a5f-43e1-a81c-9aee919243b3)

**Comment by stnguyen90** [comment]
The fix for this should be released in 1.5.6 which has https://github.com/appwrite/appwrite/pull/8080
